app/models/school.rb

```diff
@@ -1,11 +1,11 @@
 class School < ActiveRecord::Base
-  attr_accessible :name, :address, :city, :state, :bus_list_id
+  #attr_accessible :name, :address, :city, :state, :bus_list_id
```

Comment:
We can't stop using `attr_accessible` in the models without enabling [strong_parameter logic](https://github.com/rails/strong_parameters). This could cause a lot of security issues (any POST/PUT/UPDATE request can update any of the parameters otherwise, and since our source is publicized, people can know exactly what parameters to change).